src/compass/utils/lut.py

```diff
@@ -103,18 +114,230 @@ def compute_geocoding_correction_luts(burst, dem_path,
         in seconds as the function of the azimuth time and slant range.
         This correction needs to be added to the SLC tagged azimuth time to
         get the corrected azimuth times.
+
+    [rg_set, az_se]: list, np.ndarray
+        List of numpy.ndarray containing SET in slant range and azimuth directions
+        in meters. These corrections need to be added to the slC tagged azimuth
+        and slant range times.
     '''
+
+    # Some ancillary inputs
+    dem_raster = isce3.io.Raster(dem_path)
+    epsg = dem_raster.get_epsg()
+    proj = isce3.core.make_projection(epsg)
+    ellipsoid = proj.ellipsoid
+
+    # Create directory to store SET temp results
+    output_path = f'{scratch_path}/corrections'
+    os.makedirs(output_path, exist_ok=True)
+
+    # Compute Geometrical Steering Doppler
     geometrical_steering_doppler = \
         burst.doppler_induced_range_shift(range_step=rg_step, az_step=az_step)
 
+    # Compute bistatic delay
     bistatic_delay = burst.bistatic_delay(range_step=rg_step, az_step=az_step)
 
-    if not os.path.exists(dem_path):
-        raise FileNotFoundError(f'Cannot find the dem file: {dem_path}')
-
+    # Compute azimuth FM-rate mismatch
     az_fm_mismatch = burst.az_fm_rate_mismatch_mitigation(dem_path,
                                                           scratch_path,
                                                           range_step=rg_step,
                                                           az_step=az_step)
 
-    return geometrical_steering_doppler, bistatic_delay, az_fm_mismatch
+    # Get solid Earth tides on a very coarse grid
+    # Run rdr2geo on a very coarse grid
+    compute_rdr2geo_rasters(burst, ellipsoid, dem_raster, output_path,
+                            rg_step * 10, az_step * 10)
+
+    # Open rdr2geo layers and feed them to SET computation
+    lat, _ = open_raster(f'{output_path}/y.rdr')
+    lon, _ = open_raster(f'{output_path}/x.rdr')
+    inc_angle, _ = open_raster(f'{output_path}/incidence_angle.rdr')
+    head_angle, _ = open_raster(f'{output_path}/heading_angle.rdr')
+
+    # compute Solid Earth Tides (using pySolid)
+    rg_set_temp, az_set_temp = solid_earth_tides(burst, lat, lon,
+                                                 inc_angle, head_angle)
+
+    # Resize SET to the size of the correction grid
+    out_shape = bistatic_delay.data.shape
+    kwargs = dict(order=1, mode='edge', anti_aliasing=True,
+                  preserve_range=True)
+    rg_set = resize(rg_set_temp, out_shape, **kwargs)
+    az_set = resize(az_set_temp, out_shape, **kwargs)
```

Comment:
In line with what @scottstanie's [comment](https://github.com/opera-adt/COMPASS/pull/91/files#r1111324064), unless there's utility elsewhere, the lines preceding and following the call the `solid_earth_tides()` could also be in `solid_earth_tides()`.

Reply:
Uhm, we will be using the rdr2geo layers and the corresponding function that computes them for the computation of the troposphere delay using weather model. Probably, we can wait for that part of code to come out and then get back to this comment?